.github/workflows/examples.yml

```
@@ -53,9 +53,9 @@ jobs:
         if [ ${{ matrix.python-version }} = 3.6 ]; then
           IGNORES='chainermn_.*|fastai*|pytorch_distributed_.*|rapids_.*|botorch_.*|pytorch/pytorch_checkpoint*|allennlp*'
```

ditto.
AllenNLP was excluded by #2442 when PyTorch==1.8.0 was introduced. I think we can activate the AllenNLP related tests in Python 3.6 now.

```suggestion
          IGNORES='chainermn_.*|fastai*|pytorch_distributed_.*|rapids_.*|botorch_.*|pytorch/pytorch_checkpoint*'
```